test/test_reconstruction.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/python
+"""
+Details about the Doument: ///// COMPLETE
```

Comment:
is this or reconstructin-test supposed to be here?    Seems to be a duplication.

Reply:
Yes in the branch I totally removed the previous file and wrote the code
again.